package.json

```diff
@@ -31,8 +32,10 @@
     "test:typings": "npm run generate:typings && node tests/typings/copy && npm run ts && npm run clean:tests",
     "test:coverage": "jest --coverage",
     "test:smoke": "babel-node ./tests/smoke.runner.js",
+    "test:e2e": "jest --config e2e/jest.config.js --detectOpenHandles",
```

Comment:
Detect open handles prevents from running tests in parallel, I think it can be removed as far as the root cause has been fixed

Reply:
done